app/uk/gov/hmrc/agentsexternalstubs/models/RegexPatterns.scala

```diff
@@ -31,6 +32,8 @@ object RegexPatterns {
     "^((?!(BG|GB|KN|NK|NT|TN|ZZ)|(D|F|I|Q|U|V)[A-Z]|[A-Z](D|F|I|O|Q|U|V))[A-Z]{2})\\s?\\d{2}\\s?\\d{2}\\s?\\d{2}\\s?[A-D]?$".r)
   val validArn: Matcher = validate("^[A-Z]ARN[0-9]{7}$".r)
   val validUtr: Matcher = validate("^[0-9]{10}$".r)
+  val validUrn: Matcher = validate("^([A-Z0-9]{1,15})$".r)
+  val validUtrOrUrn = validateTrustTaxIdentifier("^[0-9]{10}$".r, "^([A-Z0-9]{1,15})$".r)
```

Comment:
where do we use it? not sure we need this

Reply:
we use this in DESstubcontroler line 454